client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -95,28 +99,36 @@ const SettingsItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
 	);
 };
 
-const SiteMonitoringItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
+const SiteMonitoringItem = ( { site, isWpAdminInterface, recordTracks }: SitesMenuItemProps ) => {
 	const { __ } = useI18n();
 
 	return (
 		<MenuItemLink
 			href={ getSiteMonitoringUrl( site.slug ) }
 			onClick={ () => recordTracks( 'calypso_sites_dashboard_site_action_site_monitoring_click' ) }
 		>
-			{ __( 'Site monitoring' ) }
+			{ isWpAdminInterface ? __( 'Monitoring' ) : __( 'Site monitoring' ) }
 		</MenuItemLink>
 	);
 };
 
-const ManagePluginsItem = ( { site, recordTracks }: SitesMenuItemProps ) => {
+const ManagePluginsItem = ( {
+	site,
+	isWpAdminInterface,
+	wpAdminUrl,
+	recordTracks,
+}: SitesMenuItemProps ) => {
 	const { __ } = useI18n();
 	const hasManagePluginsFeature =
 		useSelector( ( state ) => siteHasFeature( state, site.ID, WPCOM_FEATURES_MANAGE_PLUGINS ) ) ||
 		isNotAtomicJetpack( site );
 	// If the site can't manage plugins then go to the main plugins page instead
 	// because it shows an upsell message.
 	const [ href, label ] = hasManagePluginsFeature
-		? [ getManagePluginsUrl( site.slug ), __( 'Manage plugins' ) ]
+		? [
+				isWpAdminInterface ? `${ wpAdminUrl }plugins.php` : getManagePluginsUrl( site.slug ),
```

Comment:
For this one, I think we don't need the `layout/dotcom-nav-redesign` flag check. This should be true for any site with classic view enabled. We're fixing a bug here.

What do you think of putting the logic in the `getManagePluginsUrl()` instead? In the file, we already have a `siteDefaultInterface()` function that checks for the `wpcom_admin_interface` option.

Reply:
I'd prefer to keep the current way as I don't want to check whether the interface is wp-admin and the flag is enabled here and there for now 🙈

But I agree that we can refactor it to centralize the logic of getting the label and link of the menu items later 🙂